packages/ai-jsx/src/core/error-boundary.ts

```diff
@@ -5,9 +5,11 @@ import { Node, RenderContext } from '../index.js';
  *
  * If the fallback itself throws an error, that error will be propagated. (Just like if your `catch` block throws an error.)
  *
- * ```
+ * @example
+ * ```tsx
  *  <ErrorBoundary fallback="User data could not be fetched.">
  *    {fetchUserData()}
+ *
```

Comment:
Hmm, do we want this? Looks a bit funny to me.

![image](https://github.com/fixie-ai/ai-jsx/assets/829827/eea2d900-cad1-4d97-a63d-d5c2f5b4739c)

Also it's weird to me that "Example" is in code formatting, but not worth spending more than 30 seconds thinking about. 
![image](https://github.com/fixie-ai/ai-jsx/assets/829827/ebd2be9a-fdc0-4f7f-a947-07dcb4e38076)

https://ai-jsx-docs-git-mdw-api-docs-fixie-ai.vercel.app/api/modules/core_error_boundary

Reply:
That is how typedoc renders the `@example` tag. I think it's useful to have an example.